framework/doc/content/source/bcs/DirichletBC.md

```diff
@@ -33,6 +33,12 @@ before the solve begins. With `preset = false`, the boundary condition is
 only enforced as the solve progresses. In most situations, presetting the boundary
 condition is better.
 
+## Similar Dirichlet BCs
+
+To use a Function instead of a constant value : [FunctionDirichletBC.md]
+To impose a Dirichlet BC using a penalty method or on an elemental variable : [PenaltyDirichletBC.md]
+To use a another variable instead of a constant value : [MatchedValueBC.md]
```

Comment:
```suggestion
- To use a Function instead of a constant value : [FunctionDirichletBC.md]
- To impose a Dirichlet BC using a penalty method or on an elemental variable : [PenaltyDirichletBC.md]
- To use a another variable instead of a constant value : [MatchedValueBC.md]
```